### PR TITLE
Enhancement/skale 3172 timing

### PR DESCRIPTION
**Pull request by DimaStebaev**
Disable 2 epoch in September and change time parameters.

### CODE REVIEW

File: contracts/ConstantsHolder.sol [hunk 1]
```diff
@@ -110,7 +110,7 @@ contract ConstantsHolder is Permissions {
 
     uint public limitValidatorsPerDelegator;
 
-    uint public firstDelegationsMonth;
+    uint256 public firstDelegationsMonth; // deprecated
```
Comment (payvint) [inline contracts/ConstantsHolder.sol:113]: Have we already upgraded mainnet with this variable?

File: contracts/delegation/TimeHelpers.sol [hunk 2]
```diff
@@ -35,9 +35,6 @@ contract TimeHelpers {
     using SafeMath for uint;
 
     uint constant private _ZERO_YEAR = 2020;
-    
-    uint constant private _FICTIOUS_MONTH_START = 1599523200;
```
Comment (payvint) [inline contracts/delegation/TimeHelpers.sol:39]: So we did not upgrade mainnet with these vars? Does it safe to remove vars after deploying?

**Review by payvint — COMMENTED** [review]
Could you clarify some questions?

File: contracts/ConstantsHolder.sol (hunk 1 above)
Comment (DimaStebaev) [reply contracts/ConstantsHolder.sol:113]: Yes. We did.

File: contracts/delegation/TimeHelpers.sol (hunk 2 above)
Comment (DimaStebaev) [reply contracts/delegation/TimeHelpers.sol:39]: Yes. Because constant variables is stored in bytecode and are changed after upgrade.

Comment (payvint) [reply contracts/delegation/TimeHelpers.sol:39]: Nice!

File: contracts/ConstantsHolder.sol (hunk 1 above)
Comment (payvint) [reply contracts/ConstantsHolder.sol:113]: Good!

**Review by payvint — APPROVED** [review]
Awesome!!!

**Pull request merged**